tests/test_schemes.py

```diff
@@ -99,6 +101,19 @@ def test_failed_connection(self):
         scheme = Scheme('wlan0', 'test')
         self.assertRaises(ConnectionError, scheme.parse_ifup_output, FAILED_IFUP_OUTPUT)
 
+    def test_activate_is_called_with_good_args(self):
+        args = ['sudo', '/sbin/ifdown', 'wlan0', 'wlan0=wlan0-test']
+        kwargs = {'stderr':subprocess.STDOUT}
+        scheme = Scheme('wlan0', 'test')
+        subprocess.check_output = MagicMock(return_value=SUCCESSFUL_IFUP_OUTPUT)
```

Comment:
You are right. I did not pay attention but do not you think that mocking check_output everywhere would be better than calling the terminal in tests?